src/constants.ts

```diff
@@ -1 +1,2 @@
-export const CACHE_FOLDER = './node_modules/.cache/jest-preview';
+export const CACHE_FOLDER = './node_modules/.cache/jest-preview-dom';
```

Comment:
> I think it should be jest-preview instead of jest-preview-dom.

Yeah, it should be `jest-preview`

> Anyway, can you do a rebase on main instead of merge (if it does not require much effort)?

Let my update